Components/Basic/custom.component.yaml.meta.yaml

```diff
@@ -1,7 +1,33 @@
 ---
 docstring: |
     ## custom.component.yaml
```

Comment:
Titles shouldn't have the .yaml extension. It's confusing. We can revisit when we do full flow examples.